CITATION.cff

```diff
@@ -0,0 +1,25 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: pixi
+message: >-
+  If pixi contributes to a project that leads to a
+  scientific publication, please acknowledge this fact by
+  citing according to this file.
+type: software
+authors:
+  - given-names: pixi open-source contributors
```

Comment:
While I understand the idea to want to give credit to everyone who contributed, I would recommend adding the names of people who have "responsibility" for `pixi` within prefix.dev here.